### PR TITLE
chore(SidePanel): fix a11y contrast issue

### PR DESCRIPTION
**Pull request by krisantrobus**
Fix issues that occur with a11y color-contrast on Side Panel

### CODE REVIEW

File: packages/paste-core/components/side-panel/stories/index.stories.tsx [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
   SidePanel,
   SidePanelBody,
   SidePanelButton,
   SidePanelContainer,
+  SidePanelContext,
   SidePanelHeader,
   SidePanelHeaderActions,
   SidePanelPushContentWrapper,
-} from "../src";
+} from "@twilio-paste/side-panel";
```
Comment (nkrantz) [inline packages/paste-core/components/side-panel/stories/index.stories.tsx:27]: question: why this change?

File: packages/paste-core/components/side-panel/stories/index.stories.tsx [hunk 2]
```diff
@@ -1,28 +1,36 @@
+import { Anchor } from "@twilio-paste/anchor";
+import { Avatar } from "@twilio-paste/avatar";
```
Comment (nkrantz) [inline packages/paste-core/components/side-panel/stories/index.stories.tsx:2]: suggestion(non-blocking): probably best to keep these eslint-disables at the top of the file so eslint doesn't get angry about these 2 imports

File: packages/paste-core/components/side-panel/stories/index.stories.tsx (hunk 1 above)
Comment (krisantrobus) [reply packages/paste-core/components/side-panel/stories/index.stories.tsx:27]: I reviewed the other stories and they all used the same import format. I only now realize that is for all other components except the package it is changing. I will revert this

File: packages/paste-core/components/side-panel/stories/index.stories.tsx (hunk 2 above)
Comment (krisantrobus) [reply packages/paste-core/components/side-panel/stories/index.stories.tsx:2]: Great suggestion, addressed in latest PR

**Review by krisantrobus — COMMENTED** [review]
👏 it's resolved! Great final fix for this

**Review by nkrantz — APPROVED** [review]
